dynamic-property-jackson/src/main/java/ru/fix/dynamic/property/jackson/JacksonDynamicPropertyMarshaller.java

```diff
@@ -24,15 +27,16 @@
  */
 public class JacksonDynamicPropertyMarshaller implements DynamicPropertyMarshaller {
 
-    private final StdSerializer stdSerializer = new StdSerializer();
+    private final List<InternalMarshaller> marshallers;
```

Comment:
Seems like we are mixing two responsibilities into single class: 
1. Ability to compose several user defined marshallers into one. 
2. Implement default marshalling logic via Jackson serialization. 
Suggestion:
- In order to minimize confusion lest leave `DynamicPropertyMarshaller` interface as is. 
  And provide ComposableSerializer interface with methods with Optional<> results (rename InternalMarshaller to ComposableSerializer). 
-  Make class `ComposableDynamicPropertyMarshaller(List<ComposableSerializer>) implements DynamicPropertyMarshaller`   that contains list of user defined ComposableSerializers
- Make StdSerializer implement ComposableSerializer as you did.
- Move jackson mapper based serialization logic from JacksonDynamicPropertyMarshaller to JacksonSerializer and made it implement ComposableSerializer interface. (rename JacksonDynamicPropertyMarshaller to JacksonSerializer)

Now we can create our MarshallerBuilder that crate ComposableDynamicPropertyMarshaller  based on 
- User defined ComposableSerializers, StdSerializer and JacksonSerializer.

Reply:
- Renamed `InternalMarshaller` to `ComposableSerializer`
- Created `JacksonSerializer`
- Renamed `JacksonDynamicPropertyMarshaller` to `ComposableDynamicPropertyMarshaller` and refactored.